pkg/monitoring/metrics.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright The Kubernetes NMState Authors.
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package monitoring
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	ApplyTopologyTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "nmstate_apply_topology_total",
+		},
+		[]string{"name"},
+	)
+
+	ApplyFeaturesTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "nmstate_apply_features_total",
```

Comment:
@qinqon You are missing a help text for the metrics.

Reply:
If I understand currently, This metric counts the number of times the features were applied.
If that is correct I would name it `nmstate_features_applied_total`.